### DISCUSSION TITLE
Preparing for Quarkus 3

### DISCUSSION THREAD

**Discussion by maxandersen** [quarkiverse/quarkiverse#100] [Team Posts]
Hi Quarkiverse maintainers and contributors.

As some of you might have seen I last week posted ["Road to Quarkus 3" on quarkus-dev ](https://groups.google.com/g/quarkus-dev/c/xjHIBk6Mx7k).

We've done major bumps in the past, this one is though a bit different due to the Jakarta package rename meaning we expect most extensions will need to bump their own major version and keep branch that works for 2.x and move main development towards 3.x.

This is a post to increase the likelihood of you being aware Quarkus 3 is coming
and we would like to hear back from you if ready or there are concerns/questions.

Will be great to have list of Quarkiverse extensions that we can include in Quarkus 3 announce as being ready/preparing.

You can respond on the quarkus-dev post, in this thread or on zulip if you prefer chat.

**Comment by hbelmiro** [comment]
Hi @maxandersen.
I'm happy to share that https://github.com/quarkiverse/quarkus-openapi-generator is ready for Quarkus 3.

cc @ricardozanini
